dnfdaemon-server/services/rpm/rpm.cpp

```diff
@@ -24,14 +24,20 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 #include <libdnf/rpm/package_query.hpp>
 #include <libdnf/rpm/package_set.hpp>
+#include <sdbus-c++/Message.h>
```

Comment:
This is not needed - `#include <sdbus-c++/sdbus-c++.h>` on the following line takes care of it and includes all sdbus-c++ headers. See https://github.com/Kistler-Group/sdbus-cpp/blob/master/include/sdbus-c%2B%2B/sdbus-c%2B%2B.h